src/main/java/seedu/budgetbuddy/Storage.java

```diff
@@ -361,7 +362,36 @@ public void saveSplitExpenses(List<SplitExpense> splitExpenses) throws IOExcepti
                 splitExpense.getDescription()));
         }
         writer.close();
-    }    
+    }
+
+    public List<Budget> loadBudgets() throws FileNotFoundException {
+        File file = new File(filePath);
+        Scanner scanner = new Scanner(file);
+        List<Budget> loadedBudgets = new ArrayList<>();
+
+        while (scanner.hasNextLine()) {
+            String line = scanner.nextLine();
+            String[] parts = line.split("\\|");
+            String category = parts[0].trim();
+            double budgetAmount = Double.parseDouble(parts[1].trim());
+            Budget budget = new Budget(category, budgetAmount);
+            loadedBudgets.add(budget);
+        }
+
+        scanner.close();
+        return loadedBudgets;
```

Comment:
loadBudgets() method is well-programmed and code is easy to understand! Would it be better to use try and catch blocks in this method? When an exception is caught, the catch block can be used to log detailed information about the error such as including an error message.